### ISSUE TITLE
powermon errors on SNAP6

### ISSUE THREAD

**Issue by marinanderson** [ovro-lwa/lwa-issues#102]
The following power monitoring error previously appeared on snap06 and snap09:

`ERROR - powermon - Failed to instantiate I2C control object`

Mark re-seated the SD cards on both boards, then we rebooted the boards via power cycling of the SNAP chassis.

Fixed things for snap09, but still getting that same error on snap06:
`2022-04-06 13:55:19,827 - lwa_f.blocks.block:snap06 - ERROR - powermon - Failed to instantiate I2C control object`

**Comment by rh-codebase** [comment]
Perhaps try swapping the SD cards between the 2 SNAPS and see if the issue follows the card.

**Comment by Princemongo** [comment]
Mark here - were both chassis power cycled? SNAP06 is in chassis #1 while SNAP09 is in chassis #2.

**Comment by marinanderson** [comment]
Yup, we power cycled both chassis.

**Comment by Princemongo** [comment]
@jack-h Can the microSD cards be removed and inserted with the power on? Also, can the microSD cards be swapped between SNAPs?

**Comment by jack-h** [comment]
On Thu, 7 Apr 2022, 00:41 Princemongo, ***@***.***> wrote:

> @jack-h <https://github.com/jack-h> Can the microSD cards be removed and
> inserted with the power on?
>
They can be inserted and removed, but the firmware they contained is only
copied at boot, so a power cycle is required after inserting a new card.


Also, can the microSD cards be swapped between SNAPs?
>

Yes!

**Comment by Princemongo** [comment]
Has snap06 magically fixed itself or is there still a problem?

**Comment by Princemongo** [comment]
I took the microSD cards from snap06 and snap05 and when I grabbed the microSD-to-USB adapter, it had an identical card in it! I copied the bin file from snap05 by putting it on my Windows box and then overwriting the files on the other two microSD cards. When I inserted them back into the snaps, now neither works! I'm thinking it might be a mistake to use Windows. I will repeat the process tomorrow but using the microSD card from snap04 as the "master".
